src/utils/compose_ts.ts

```diff
@@ -1292,7 +1292,7 @@ async function pushAndUpdateServiceImages(
 				localImage.inspect(),
 				pushImage(localImage, index),
 			]);
-			serviceImage.image_size = imgInfo.Size;
+			serviceImage.image_size = imgInfo.Size.toString();
```

Comment:
NIT: I think that the latest balenaLint might convert this to v
```suggestion
			serviceImage.image_size = `${imgInfo.Size}`;
```